pynxtools/dataconverter/readers/em/geometry/convention_mapper.py

```diff
@@ -0,0 +1,83 @@
+#
+# Copyright The NOMAD Authors.
+#
+# This file is part of NOMAD. See https://nomad-lab.eu for further info.
+#
+# Licensed under the Apache License, Version 2.0 (the "License");
+# you may not use this file except in compliance with the License.
+# You may obtain a copy of the License at
+#
+#     http://www.apache.org/licenses/LICENSE-2.0
+#
+# Unless required by applicable law or agreed to in writing, software
+# distributed under the License is distributed on an "AS IS" BASIS,
+# WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+# See the License for the specific language governing permissions and
+# limitations under the License.
+#
+"""Parse conventions from an ELN schema instance."""
+
+# pylint: disable=no-member
+
+# ,duplicate-code
+
+from pynxtools.dataconverter.readers.em.geometry.handed_cartesian \
+    import REFERENCE_FRAMES, AXIS_DIRECTIONS, is_cs_well_defined
+
+from pynxtools.dataconverter.readers.em.concepts.concept_mapper \
+    import variadic_path_to_specific_path, apply_modifier
+
+from pynxtools.dataconverter.readers.em.geometry.geometry \
+    import NxEmConventions
+
+# example how to check against different types of Euler angle conventions
+# from pynxtools.dataconverter.readers.em.geometry.euler_angle_convention \
+#    import which_euler_convention
+
+# example how to check if set of conventions matches to some suggestion in the literature
+# from pynxtools.dataconverter.readers.em.geometry.msmse_convention \
+#     import is_consistent_with_msmse_convention
```

Comment:
Please place this in a better place and remove it from here.

Reply:
removed in pynxtools-em